docs/pubsub-usage.rst

```diff
@@ -20,24 +32,27 @@ Create a new topic for the default project:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> topic = Topic('topic_name')
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
    >>> topic.create()  # API request
 
-Create a new topic for an explicit project:
+Create a new topic for an explicit client:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> topic = Topic('topic_name', project='my.project')
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = Topic('topic_name', client=client)
```

Comment:
Yeah I was just trying to map to what was previously there. I'm happy to nuke the section.